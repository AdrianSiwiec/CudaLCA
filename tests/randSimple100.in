100
0
0
2
1
0
0
0
0
8
1
2
10
12
9
3
15
8
13
11
13
16
10
1
6
4
2
15
11
0
27
29
19
6
32
16
16
0
8
8
9
11
18
40
35
35
13
27
21
26
37
22
8
34
26
2
19
5
41
24
57
27
6
25
25
25
44
49
62
35
19
9
35
8
59
70
57
21
45
29
30
5
78
19
66
43
83
32
53
32
51
15
77
10
76
89
4
88
66
15
100
10 36
67 94
93 11
80 37
75 93
55 94
51 50
12 29
91 29
6 14
7 4
33 12
70 15
27 12
53 88
24 82
76 63
26 43
33 21
14 7
53 8
58 21
50 4
86 22
37 93
49 44
57 96
63 83
47 27
68 91
25 52
15 44
88 53
74 42
1 21
81 88
61 54
11 91
13 63
57 97
93 2
99 58
41 50
30 14
5 41
93 98
94 82
35 61
55 82
55 9
49 56
62 36
27 10
74 73
22 41
95 83
41 16
18 46
61 35
76 0
98 18
1 21
82 45
79 88
98 37
93 35
23 99
62 8
81 3
96 88
71 4
72 91
38 64
51 42
43 99
17 27
1 93
38 70
59 83
73 18
53 9
8 18
26 28
31 22
10 8
64 27
19 33
98 88
30 9
8 1
81 73
67 77
48 82
65 5
23 59
68 90
9 28
57 28
51 35
95 40
