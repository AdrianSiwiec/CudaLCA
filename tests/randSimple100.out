0
0
0
0
0
0
0
0
0
0
0
0
2
0
53
0
0
0
0
0
0
2
0
0
0
2
0
0
27
0
0
15
53
0
0
0
2
2
1
0
0
2
0
0
0
0
0
15
2
0
2
0
0
0
0
19
2
13
15
0
0
0
45
0
0
0
0
0
0
0
0
15
0
10
0
0
1
0
0
0
0
0
2
0
0
0
0
2
0
0
0
2
16
0
0
0
0
0
0
0
